docs/tactics.md

```diff
@@ -64,6 +64,19 @@ All accept an optional list of simplifier rules, typically definitions that shou
 Evaluate expressions in the language of (semi-)rings.
 Based on [Proving Equalities in a Commutative Ring Done Right in Coq](http://www.cs.ru.nl/~freek/courses/tt-2014/read/10.1.1.61.3041.pdf) by Benjamin Grégoire and Assia Mahboubi.
 
+### pi_instance
```

Comment:
I don't think this should be documented here, since it's not an interactive tactic (and it shouldn't be). The documentation at `pi_instance` is fine.

Reply:
It isn't part of the standard set of tactics that mathlib supplies, which is what this page is for. It's a special use tactic, so the file that uses it is better documentation to its use than a general purpose list intended for people learning to use the tactic language. It is comparable to other "local tactics" like `num.transfer` which are defined and used on the spot.